src/commands/deployAzurePipeline/configurePipeline.ts

```diff
@@ -0,0 +1,50 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void | undefined> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
+    if (!deployToAzureExtensionInstalled) {
+        return undefined;
+    }
+
+    const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+    if (!cloudExplorer.available) {
+        return undefined;
+    }
+
+    const clusterTarget = cloudExplorer.api.resolveCommandTarget(target);
+    if (clusterTarget && clusterTarget.cloudName === "Azure" && clusterTarget.nodeType === "resource" && clusterTarget.cloudResource.nodeType === "cluster") {
+        const configurePipelineCommand = 'configure-cicd-pipeline';
```

Comment:
Consider pulling this out to a top-level constant (in `SHOUTY_CAPS`) so it jumps out as a true 'magic string' constant rather than just an immutable variable.

Reply:
💡 I believe it means moving this back to the top level with : sample: `const MY_CONSTANT = "blah";` 

* Please note you need to make sure this is well documented in the your extension so that the command line et. al. don't change and if it does people should be aware of this.

Reply:
@Tatsinnit we deliberately moved it to function level as it is being used only in that particular function. Just trying to understand the reason behind moving it back at the top level?

Reply:
Thanks, @kanika1894 this one I totally remember so here is my take: I am easy with any favour but remember what you had earlier was all lower casing: example: https://github.com/Azure/vscode-aks-tools/pull/42/commits/5fe6ea16e4796c079a7fbeab9f64a9bf6c0d087d#diff-5b5e105bbccdadd756c4442822456a4bR7

Like I have iterated in past I am easy with whatever According to what code design needs to look like and I can see making all in CAPS here kind of highlights the fact that hey these are key Const. so make sense to do it that way. What do you say?

(Just to keep as side note) Also please note we should highlight in the parent extension that all these constant which are actually DTA extension dependencies cannot be changed as it will break these functionality.